packages/reports/addon/templates/components/common-actions/schedule.hbs

```diff
@@ -87,7 +87,7 @@
             size="small"
             class="schedule-modal__must-have-data-toggle"
             value=localDeliveryRule.schedulingRules.mustHaveData
-            onToggle=(mut localDeliveryRule.schedulingRules.mustHaveData)}}
+            onToggle=(action "toggleMustHaveData")}}
```

Comment:
Can we use `toggle` instead? https://github.com/DockYard/ember-composable-helpers#toggle